bin/copy.js

```diff
@@ -47,6 +46,8 @@ async function copy(source, destination, watch = false) {
   let scheduled = undefined
 
   if (watch === true) {
+    // Dynamically import chokidar only when watch is true
```

Comment:
The comment is kinda weird without context. I suggest either removing it or adding info about why we shouldn't use static imports.